chapters/annotations.tex

```diff
@@ -1197,10 +1193,10 @@ \subsubsection{Ellipse}\label{ellipse}
 These specify the endpoints of the arc prior to the stretch and rotate operations.
 The arc is drawn counter-clockwise from \lstinline!startAngle! to \lstinline!endAngle!, where \lstinline!startAngle! and \lstinline!endAngle! are defined counter-clockwise from 3 o'clock (the positive x-axis).
 
-The closure attribute specifies whether the endpoints specified by \lstinline!startAngle! and \lstinline!endAngle! are to be joined by lines to the center of the extent (\lstinline!closure = EllipseClosure.Radial!), joined by a single straight line between the end points (\lstinline!closure = EllipseClosure.Chord!), or left unconnected (\lstinline!closure = EllipseClosure.None!).
+The \lstinline!closure! attribute specifies whether the endpoints specified by \lstinline!startAngle! and \lstinline!endAngle! are to be joined by lines to the center of the extent (\lstinline!closure = EllipseClosure.Radial!), joined by a single straight line between the end points (\lstinline!closure = EllipseClosure.Chord!), or left unconnected (\lstinline!closure = EllipseClosure.None!).
 In the latter case, the ellipse is treated as an open curve instead of a closed shape, and the \lstinline!fillPattern! and \lstinline!fillColor! are not applied (if present, they are ignored).
 
-The default closure is \lstinline!EllipseClosure.Chord! when \lstinline!startAngle! is 0 and \lstinline!endAngle! is 360, or \lstinline!EllipseClosure.Radial! otherwise.
+The effect of \lstinline!EllipseClosure.Default! is that of \lstinline!EllipseClosure.Chord! when both \lstinline!startAngle! is 0 and \lstinline!endAngle! is 360, and that of \lstinline!EllipseClosure.Radial! otherwise.
```

Comment:
```suggestion
The effect of \lstinline!EllipseClosure.Automatic! is that of \lstinline!EllipseClosure.Chord! when both \lstinline!startAngle! is 0 and \lstinline!endAngle! is 360, and that of \lstinline!EllipseClosure.Radial! otherwise.
```
according to poll

Reply:
Done with all renaming in a single commit.  Resolving.